src/printer.js

```diff
@@ -1335,6 +1335,21 @@ function printLines(path, options, print, childrenAttribute = "children") {
     }
 
     if (isInlineNode) {
+      const printComments = comments => {
```

Comment:
Let's move this function to `comments.js`

Reply:
Good idea :+1: Also refactored a little.